cmd/gossamer/configcmd.go

```diff
@@ -53,11 +54,8 @@ func makeNode(ctx *cli.Context) (*dot.Dot, *cfg.Config, error) {
 
 	var srvcs []services.Service
 
-	// DB: Create database dir and initialize stateDB and blockDB
-	dbSrv, err := polkadb.NewDbService(fig.Global.DataDir)
-	if err != nil {
-		return nil, nil, fmt.Errorf("cannot create db service: %s", err)
-	}
+	// DB: Create service, initialize stateDB and blockDB
+	dbSrv := state.NewService(fig.Global.DataDir)
```

Comment:
perhaps rename this to `stateSrv` for clarity?

Reply:
good idea